docs/kusion/5-user-guides/1-cloud-resources/1-database.md

```diff
@@ -63,133 +156,58 @@ Enter a value or leave blank to accept the (default), and press <ENTER>.
-                    "WORDPRESS_DB_NAME": "mysql"
-                }
-                resources: {
-                    "cpu": "500m"
-                    "memory": "512Mi"
-                }
-            }
-        }
-        replicas: 1
-        ports: [
-            n.Port {
-                port: 80
-            }
-        ]
-    }
-    database: db.Database {
-        type: "aws"
-        engine: "MySQL"
-        version: "8.0"
-        size: 20
-        instanceType: "db.t3.micro"
-    }
-}
+1 directory, 3 files
 ```
 
-`dev/platform.k`
-```python
-import catalog.models.schema.v1 as ac
-
-# platform.k declares customized configurations
-wordpress: ac.AppConfiguration {
-    database: {
-        privateRouting = False
-
-        # SubnetID defines the virtual subnet ID associated with the VPC that the rds
-        # instance will be created in. The rds instance won't be created in user's own VPC
-        # but in default VPC of cloud vendor, if this field is not provided.
-        # subnetID = [your-subnet-id]
+:::info
+More details about the directory structure can be found in [Project](../../3-concepts/1-project/1-overview.md) and [Stack](../../3-concepts/2-stack/1-overview.md). 
+:::
 
-        # category = "serverless_basic"
-    }
-}
-```
+### Review Configuration Files
 
-The template project defaults to use `AWS` RDS instance for the WordPress application. If you would like to try creating the `Alicloud` RDS instance, you could modify the `dev/main.k` and `dev/platform.k` to the following and replace the `[your-subnet-id]` in `dev/platform.k` with the `vSwitchID` to provision the database in. 
+Now let's take a look at the configuration files located in `dev/main.k`.
 
-`dev/main.k`
 ```python
 import catalog.models.schema.v1 as ac
-import catalog.models.schema.v1.trait as t
 import catalog.models.schema.v1.workload as wl
 import catalog.models.schema.v1.workload.container as c
-import catalog.models.schema.v1.workload.container.probe as p
-import catalog.models.schema.v1.workload.secret as sec
 import catalog.models.schema.v1.workload.network as n
-import catalog.models.schema.v1.monitoring as m
-import catalog.models.schema.v1.accessories.database as db
+import catalog.models.schema.v1.accessories.mysql
 
-# main.k declares reusable configurations for dev stacks.
+# main.k declares customized configurations for dev stacks.
 wordpress: ac.AppConfiguration {
     workload: wl.Service {
         containers: {
             wordpress: c.Container {
-                image = "wordpress:6.3"
+                image: "wordpress:6.3"
                 env: {
-                    "WORDPRESS_DB_HOST": "$(KUSION_DB_HOST)"
-                    "WORDPRESS_DB_USER": "$(KUSION_DB_USERNAME)"
-                    "WORDPRESS_DB_PASSWORD": "$(KUSION_DB_PASSWORD)"
+                    "WORDPRESS_DB_HOST": "$(KUSION_DB_HOST_WORDPRESS_MYSQL)"
+                    "WORDPRESS_DB_USER": "$(KUSION_DB_USERNAME_WORDPRESS_MYSQL)"
```

Comment:
ditto

Reply:
The explanation of these magic values has been added below already.